app/controllers/comments_controller.rb

```diff
@@ -10,6 +10,7 @@ def show
 
   def create
     comment = discussion.comments.create(comment_params)
+    discussion.update_subscriptions
```

Comment:
I don't think it's the controller's job to know this. I propose that we create an object to handle this. Something like a `CommentCreator`, or `CreatesComment`, or even `Commenter`. 

This make the job of the controller much clearer and to the point. Get HTTP request to create comment, parse params, tell someone to create a comment with the params, respond accordingly.